src/typography/shared/Shared.stories.tsx

```diff
@@ -1,5 +1,6 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
+export default {};
```

Comment:
File can be renamed to `shared.ts` no?
Then the `export default` would no longer be required

Reply:
thx for the input, changing it